app/websocket/app/main/routes.py

```diff
@@ -53,7 +53,11 @@ def model_inference(model_id):
     # to the local worker in order to execute it
     sy.hook.local_worker.register_obj(data)
 
-    response = model(data).detach().numpy().tolist()
+    response = model(data)
```

Comment:
add a comment describing why this check is needed.

Reply:
Done! (b017e19)